### PR TITLE
Make ListFeed list posts from authenticated user's clubs

### PR DESCRIPTION
**Pull request by DrDab**

### CODE REVIEW

File: backend/BCAPIServer/src/main/java/routes/bookclubs/ListFeed.java [hunk 1]
```diff
@@ -1,34 +1,69 @@
...
-        // For now just return a list of all the posts.
-        List<BookPost> posts = books.listBookPosts("*");
-        for (BookPost post : posts) {
-            System.out.println(post.toString());
-        }
+    List<String> subscribedBookKeys = users.getSubscribedClubs();
+    JsonArray postsArr = new JsonArray();
 
-        return jsonObject;
+    for (String bookKey : subscribedBookKeys) {
```
Comment (Amandaha29) [inline backend/BCAPIServer/src/main/java/routes/bookclubs/ListFeed.java:59]: are we still adding posts from people you follow? or just book clubs subscribed?

Comment (DrDab) [reply backend/BCAPIServer/src/main/java/routes/bookclubs/ListFeed.java:59]: I can add that in rq

**Comment by DrDab** [comment]
@Amandaha29 How does this look? Added support for listing posts made by friends and also made it sort by  descending chronological order on the backend side

**Review by Amandaha29 — COMMENTED** [review]
lgmt

**Review by Amandaha29 — APPROVED** [review]

**Pull request merged**